core/src/main/java/org/ehcache/internal/executor/ExecutorServiceType.java

```diff
@@ -18,6 +18,7 @@
 /**
  * @author palmanojkumar
  *
+ * This is codifying the constraint possibilities.
  */
```

Comment:
We discussed other points in last conversation. Here even though it represent possible constraints but I support for this as client need specific type of executor service which they know at request time.
I will take care of provided comments by merging this pull request.